src/password-store.sh

```diff
@@ -32,7 +32,7 @@ git_set_dir() {
 	else
 		local dir="$gwt/$1"
 	fi
-	if [[ -d "$dir/.git" ]]; then
+	if [[ -d "$dir/.git" ]] || [[ -f "$dir/.git" ]]; then
```

Comment:
Why not `[[ -e "$dir/.git" ]]`?

While it's technically not equivalent, I'm pretty sure that creating a `.git` device node or something equally crazy breaks Git in a bunch of funny ways anyhow.